djaoapp/api/contact.py

```diff
@@ -111,3 +116,119 @@ def post(self, request, *args, **kwargs):
 _("Sorry, there was an issue sending your request for information"\
 " to '%(full_name)s &lt;%(email)s&gt;'.") % {
     'full_name': provider.full_name, 'email': provider.email}})
+
+
+class PlacesSuggestionsAPIView(GenericAPIView):
+    """
+    Returns address typeahead suggestions
+
+    The API is typically used within a profile page.
+
+    **Tags**: profile
+
+    **Examples
+
+    .. code-block:: http
+
+        GET /api/accounts/places?q=123 HTTP/1.1
+
+    responds
+
+    .. code-block:: json
+
+       {
+            "count": 5,
+            "results": [
+                {
+                "description": "123 William Street, New York, NY, USA",
+                "place_id": "ChIJIaGbBBhawokRUmbgNsUmr-s"
+                },
+                {
+                "description": "1230 6th Avenue, New York, NY, USA",
+                "place_id": "ChIJC8abO_9YwokRlbzMgC8_XWE"
+                },
+                {
+                "description": "1233 York Avenue, New York, NY, USA",
+                "place_id": "ChIJ1_GF_cJYwokR8u455JHGkBA"
+                },
+                {
+                "description": "123 Melrose Street, Brooklyn, NY, USA",
+                "place_id": "ChIJZbhuygdcwokR_5NT5POb8Z8"
+                },
+                {
+                "description": "1239 Broadway, New York, NY, USA",
+                "place_id": "ChIJMebitahZwokRa9MwUrkbedc"
+                }
+            ]
+        }
+    """
+    serializer_class = PlacesSuggestionsSerializer
+    pagination_class = None
+
```

Comment:
It seems that the way to add the `q` parameter in the OpenAPI doc is to create a filter and [define `get_schema_operation_parameters`](https://github.com/djaodjin/djaodjin-saas/blob/018fe9be8d34e3c11557c1e7aae8966d2a971757/saas/filters.py#L214) in it.

Based on the implementation of `get` here, it might just work to add a line like:

```
filters = (saas.filters.SearchFilter, )
```